doc/install.rst

```diff
@@ -19,44 +19,11 @@ Install from sources
 Prerequisites
 -------------
 
-The dependencies required by the RAMP bundle are the following:
-
-1. ``ramp-database``:
-    * bcrypt
-    * click
-    * gitpython
-    * nbconvert
-    * numpy
-    * pandas
-    * psycopg2-binary
-    * sqlalchemy
-2. ``ramp-engine``:
-    * click
-    * numpy
-    * psycopg2-binary
-    * sqlalchemy
-3. ``ramp-frontend``:
-    * bokeh
-    * click
-    * Flask
-    * Flask-Login
-    * Flask-Mail
-    * Flask-SQLAlchemy
-    * Flask-WTF
-    * numpy
-    * pandas
-4. ``ramp-utils``:
-    * click
-    * pandas
-    * pyyaml
```

Comment:
I don't think it is helpful to have this list in the docs. Users can check the `environment.yml` file or the individual `setup.py` files an up to date list.